cpp/CMakeLists.txt

```diff
@@ -423,6 +442,13 @@ endif(BUILD_CUML_EXAMPLES)
 # - install targets -------------------------------------------------------------------------------
 
 install(TARGETS ${CUML_CPP_TARGET} DESTINATION lib)
+install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/include/cuml
```

Comment:
Is there any chance to only install `cuml` include folder and none of the ml-prims headers?

Reply:
The only reason I had to install cuml allocator and utils.h is the need of the device and hostallocator in cuml.hpp, so haven’t quite thought of a way to avoid that. Any ideas? Also what is your reasoning to want to avoid it?

Reply:
My ideal installation for cuML is *only* the collection of headers that define the cuml interface. Because this gives us with the maximum flexibility in changing the underlying ml-prims structure and not having to worry about our end users getting affected by this change.

Is there a chance you can give me push access to your `fea-cpp-include` branch? I can try to help you with this one.

Reply:
You should have push access now :)